pkg/sideinputs/utils/utils.go

```diff
@@ -0,0 +1,52 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+)
+
+// CheckFileExists checks if a file with the given fileName exists in the file system.
+func CheckFileExists(fileName string) bool {
+	_, err := os.Stat(fileName)
+	// check if err is "file not exists"
+	if os.IsNotExist(err) {
+		return false
+	} else {
+		return true
+	}
+}
+
+// UpdateSideInputStore writes the given value to the side-input file specified.
+func UpdateSideInputStore(filePath string, value []byte) error {
```

Comment:
The naming is tricky, it updates the file not the store, change it to something like `UpdateSideInputFile`?